### PR TITLE
Improve 120m global mosaic description

### PR DESCRIPTION
**Pull request by mlubej**
Changes:
- improved description
- added field for QM band description in the README.md
- whitespace removal
- new animations
- resize images in README.md
- other small fixes

### CODE REVIEW

**Comment by mlubej** [comment]
@Dorothyrono I can't add you as a reviewer, but here is a PR with some suggested changes for the collection description. 

Is there a way to check how the new yaml renders before it goes online?

**Comment by Dorothyrono** [comment]
@mlubej thanks alot  for the improvements,

yes we do  normally check with a  local build by running  `npm run build ` which then creates an `output `directory with html pages of all collections

but first  add .env file to root  with the following contents
```
GIT_HUB_COLLECTIONS_REPO=sentinel-hub/public-collections
GIT_HUB_COLLECTIONS_BRANCH=main
COLLECTIONS_BROWSER_ROOT_URL="https://collections.sentinel-hub.com/"
COLLECTIONS_BROWSER_S3_BUCKET=s3://collections.sentinel-hub.com/

```
could you try this local build  ?

File: collections/sentinel-s2-l2a-mosaic-120.yaml [hunk 1]
```diff
@@ -56,8 +58,16 @@ BandInformation:
           Notes: DN, typically from 0-10000
         - Name: QM
           Description: QualityMask
-          Notes: DN, 0 - Ok; 1 - no data; 2 - interpolation failed
-Contact:  https://forum.sentinel-hub.com  
+          Notes: DN, 0 - no data; 1 - interpolation passed; 2 - artificially raised value; 3 - interpolation failed
+  QM (Quality Mask) Details: |
```
Comment (Dorothyrono) [inline collections/sentinel-s2-l2a-mosaic-120.yaml:62]: For now the contents for this field,  won't be displayed, as this new new field is not declared in the details.hbs file .

generally the fields declared in details.hbs are quite general and not collection specific

So I suggest we take these contents to the README.md file under band information
unless it is really neccessary to have it here, then we ll need to declare the field first

Comment (mlubej) [reply collections/sentinel-s2-l2a-mosaic-120.yaml:62]: Yes, I noticed these yaml files are some templates and that this isn't shown. I agree with your suggestion.

**Comment by mlubej** [comment]
Thanks @Dorothyrono. `npm run build` works and I checked and removed the problems. Additionally I added some animations and resized them so that they fit the screen.

File: collections/sentinel-s2-l2a-mosaic-120.yaml [hunk 2]
```diff
@@ -56,8 +56,8 @@ BandInformation:
           Notes: DN, typically from 0-10000
         - Name: QM
           Description: QualityMask
-          Notes: DN, 0 - Ok; 1 - no data; 2 - interpolation failed
-Contact:  https://forum.sentinel-hub.com  
+          Notes: DN, from 0-3; OK if larger than 0. More info under the additional info field.
```
Comment (Dorothyrono) [inline collections/sentinel-s2-l2a-mosaic-120.yaml:59]: much better to add the link  to the additional info field
here is the link  `https://collections.sentinel-hub.com/sentinel-s2-l2a-mosaic-120/readme.html`

File: collections/sentinel-s2-l2a-mosaic-120/README.MD [hunk 3]
```diff
@@ -1,23 +1,46 @@
...
+Sentinel-2 L2A 120m mosaic is a derived product, which contains best pixel values for 10-daily periods, modelled by removing the cloudy pixels and then performing interpolation among remaining values. As clouds can be missed and as there are some parts of the world which have lengthy cloudy periods, clouds might be remaining in some parts. The actual modelling script is available [here](https://sentinel-hub.github.io/custom-scripts/sentinel-2/interpolated_time_series/).
 
-## Band information  
-Sentinel-2 L2A 120m mosaic contains 12 Sentinel 2 L2A bands and 1 quality mask. Find more information about sentinel 2 L2A bands [here](https://docs.sentinel-hub.com/api/latest/data/sentinel-2-l2a/#available-bands-and-data)
+## Band information
+Sentinel-2 L2A 120m mosaic contains 12 Sentinel 2 L2A bands and 1 quality mask. Find more information about sentinel 2 L2A bands [here](https://docs.sentinel-hub.com/api/latest/data/sentinel-1-l2a/#available-bands-and-data).
```
Comment (Dorothyrono) [inline collections/sentinel-s2-l2a-mosaic-120/README.MD:8]: no idea how we now got a wrong url name  `sentinel -1-l2a `

revert back to sentinel -2-l2a

File: collections/sentinel-s2-l2a-mosaic-120.yaml (hunk 1 above)
Comment (Dorothyrono) [reply collections/sentinel-s2-l2a-mosaic-120.yaml]: thanks .. now looks good

File: collections/sentinel-s2-l2a-mosaic-120/README.MD (hunk 3 above)
Comment (mlubej) [reply collections/sentinel-s2-l2a-mosaic-120/README.MD:8]: Good catch, no idea how this happened.

File: collections/sentinel-s2-l2a-mosaic-120.yaml (hunk 2 above)
Comment (mlubej) [reply collections/sentinel-s2-l2a-mosaic-120.yaml:59]: I tried and it seems that the link is not rendered, it just shows it how it is. Should it be done in some other way?

image

Perhaps I should try with raw html

Comment (mlubej) [reply collections/sentinel-s2-l2a-mosaic-120.yaml]: Yes, raw html works. Fixed.

File: collections/sentinel-s2-l2a-mosaic-120/README.MD (hunk 3 above)
Comment (mlubej) [reply collections/sentinel-s2-l2a-mosaic-120/README.MD]: Fixed.

**Pull request merged**